src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -417,6 +420,10 @@ RelationalDBInterfaceSqliteImp::getMinLedgerSeq()
 std::optional<LedgerIndex>
 RelationalDBInterfaceSqliteImp::getTransactionsMinLedgerSeq()
 {
+    /* if Tx tables not used, return empty value */
+    if (!useTxTables_)
+        return {};
```

Comment:
Doesn't the `getMinLedgerSeq` on line 402 need this check too? Same with `getMaxLedgerSeq`?

Reply:
No. On line 402 and getMaxLedgerSeq() we look for an answer in ledger databases, but in line 423 and similar we look it in transaction databases.

Reply:
Yes, you're right. I got confused with the ledger database and the transaction database.

Reply:
Ok